pkg/ui/src/InventoryTable/InventoryTableRow.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import type { TableData } from './types';
+
+	export let rowData: TableData;
+	export let className = '';
+</script>
+
+<tr class={className}>
+	{#each Object.values(rowData) as cell, index}
```

Comment:
What happens if I, for instance omit the editor, but add quantity, would the quantity then be shown in place of editor?